app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Expense Transactions app</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+  <div class="flex-column-centered expenses ">
+    <div class="transaction-list hidden-scroll">
+      <% @expenses.each do |expense| %>
+        <%= link_to payments_path(id: expense.id), class: "transaction-name" do %>
+        <div class="transaction-row flex-centered">
+          <div class="flex-spaced">
+            <div class="flex-spaced">
+            <p><%= image_tag("#{expense.icon.downcase}.png", alt: "Back") %></p>
+              <div class="transaction-name"><%= expense.name %></div>
+            </div>
+          </div>
+          <div class="transaction-created-at">
+            <% sum = 0 %>
+            <% count = 0 %>
+            <% @payments.select { |payment| payment.transaction_id == expense.id }.each do |payment| %>
+                <% sum += payment.amount %>
+                <% count += 1 %>
+            <% end %>
+            <%= count %> Items,
+            Total $<%= sum %>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+  <div class="box w-100 fit-content">
+      <%= link_to 'Add Expense Category', new_expense_path, class: 'inner-border no-decoration new-payment-btn ' %>
+  </div>
```

Comment:
- [ ] @kalbek Great work on your categories page however, the design is supposed to follow the guidelines given and there are some mismatches between your design and the design given, The card holding the categories should be a little bigger also you can choose to shift the number of items to maybe where the date is in original design this will make your design better, kindly check the comparison below for a better understanding:


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/kalbek/Budget-app/assets/61408860/986fa574-9d85-4d3a-87c1-35d0ba8baa3d)  |  ![image](https://github.com/kalbek/Budget-app/assets/61408860/dd7f76c4-defc-496d-9dc7-5fbce89f865f)